slowfs.py

```diff
@@ -33,7 +38,7 @@ def parse_args(args):
     parser = argparse.ArgumentParser(description='A slow filesystem.')
     parser.add_argument('-c', '--config',
                         help='path to configuration file')
-    parser.add_argument('--debug', action='store_true',
+    parser.add_argument('-d', '--debug', action='store_true',
```

Comment:
Good change, but also not related to python 3 - please send another PR for rthis.